Doc/library/pathlib.rst

```diff
@@ -1633,6 +1633,34 @@ Copying, renaming and deleting
       Added return value, return the new :class:`!Path` instance.
 
 
+.. method:: Path.delete(ignore_errors=False, on_error=None)
+
+   Recursively remove this file or directory tree.
```

Comment:
One can't recursively delete a file, and an attempt to make the implications mildly clearer. I haven't checked if 'directory tree' is a very widely-used term at present, but if so it might be worth adding a definition in the glossary (e.g. does a 'tree' require files, sub folders, etc etc -- simple when you're anything more than an absolute beginner, but pathlib is enticing for beginners!)
```suggestion
   Delete this file or directory (including all sub-directories).
```